### ISSUE TITLE
Activation button on the status bar and improve the original activation button.

### ISSUE THREAD

**Issue by Daksh-DA07** [microsoft/vscode-livepreview#467]
The current activation button is not very understandable and it would be better if Microsoft made a centralized button on the status bar for switching the live server on/off like the [live server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) extension; or make the current button better by changing its logo/symbol.

**Comment by andreamah** [comment]
Hi! Thanks for the feedback! 

The reason that we don't have a toggle on/off on the status bar is because it's easy to crowd the status bar with a lot of items. If you want a more visual way to see whether the server is on/off, I'd suggest using the `Live Preview: Start Server Logging` command. This should open a terminal with server traffic, and closing it with all embedded previews closed should stop the server.

We want to push for more of an experience where users can preview files without even thinking about the server. Hence, the main way to preview is via [the preview button in the top right corner of the file](https://github.com/microsoft/vscode-livepreview#html-file-previewing)  to take after the build-in markdown previewer.

**Comment by Daksh-DA07** [comment]
Thanks, Andrea,

I understand that the status bar, but [Microsoft](https://microsoft.com) can change the start server button's logo to improve user understanding and make it an improved dual switch for switching the live server on and off.

**Comment by andreamah** [comment]
The start server button is supposed to be more of a "preview" button, so making it toggle states would be confusing.

I can make this a feature request for a persistent status bar item (this would be disabled by default). We currently have this if the server is on, but nothing if the server is off.

<img width="94" alt="image" src="https://user-images.githubusercontent.com/31675041/227992385-a5c99b22-7e05-49bd-b8f4-6458300e0fa4.png">

However, this likely won't be a priority because we try to keep the status bar minimal.

**Comment by alivault** [comment]
Agreed with @Daksh-Ag I would also like to see a status bar button indicating server status even when the server is off. Given that the similar Live Server extension, boasting over 36M downloads, incorporates such a feature, this addition could facilitate a smoother transition for users coming to Live Preview. Despite potential clutter concerns, VS Code natively allows for the concealment of status bar items. This resolves the issue while addressing the absence of a constant indicator of server status, which seems like an oversight from a Live Server extension user's perspective. Incorporating a straightforward 'Go Live' button would be a welcome improvement.
![vscode-live-server-statusbar-3](https://github.com/microsoft/vscode-livepreview/assets/48610822/6caa6d43-aeef-4261-948a-0b6774dfcd2f)

**Comment by robisadev** [comment]
I can't tell whether the server is running or not since "Ports" tells me nothing but what ports are open:
![vscode_ports](https://github.com/microsoft/vscode-livepreview/assets/10653399/38e7cbf9-a177-46a4-923c-15540fc9eb2c)

And only after I select "Live Preview: Start Live Server" do I get:
![vscode_live-preview](https://github.com/microsoft/vscode-livepreview/assets/10653399/56d3cc5b-38fe-48a8-a925-31f541171bf2)

I know one doesn't have much to do with the other it is just confusing initially. I never even noticed that second one until after reading your post since initially there is no indication that any information would be there. At least make it an option to have a toggle always in the status bar.
